static/app/views/profiling/content.tsx

```diff
@@ -34,13 +85,24 @@ function ProfilingContent({location}: ProfilingContentProps) {
             </Layout.Header>
             <Layout.Body>
               <Layout.Main fullWidth>
+                {error && (
+                  <Alert type="error" icon={<IconFlag size="md" />}>
+                    {error}
+                  </Alert>
+                )}
                 <ProfilingScatterChart
                   {...dateSelection}
-                  traces={[]}
-                  loading={false}
-                  reloading={false}
+                  traces={traces}
+                  loading={loading}
+                  reloading={loading}
+                />
+                <ProfilingTable
+                  loading={loading}
```

Comment:
I saw in some other places that we use the isLoading prop naming convention. Is that the more common style or is this the more common one?

Reply:
I don't think we've landed on a standard for this but I do think `isLoading` is more explicit so I'll update it.